src/ui/pages/Menu/Menu.tsx

```diff
@@ -0,0 +1,130 @@
+import {
+  IonButton,
+  IonCol,
+  IonIcon,
+  IonItem,
+  IonLabel,
+  IonRow,
+  useIonViewWillEnter,
+} from "@ionic/react";
+import {
+  settingsOutline,
+  personCircleOutline,
+  walletOutline,
+  peopleOutline,
+  chatbubbleOutline,
+  fingerPrintOutline,
+  idCardOutline,
+} from "ionicons/icons";
+import { TabLayout } from "../../components/layout/TabLayout";
+import { useAppDispatch } from "../../../store/hooks";
+import { setCurrentRoute } from "../../../store/reducers/stateCache";
+import { TabsRoutePath } from "../../../routes/paths";
+import "./Menu.scss";
+import { i18n } from "../../../i18n";
+
+const Menu = () => {
+  const pageId = "menu-tab";
+  const dispatch = useAppDispatch();
+
+  useIonViewWillEnter(() => {
+    dispatch(setCurrentRoute({ path: TabsRoutePath.MENU }));
+  });
+
+  const AdditionalButtons = () => {
+    return (
+      <IonButton
+        shape="round"
+        className="connections-button"
+        data-testid="connections-button"
+      >
+        <IonIcon
+          slot="icon-only"
+          icon={settingsOutline}
+          color="primary"
+        />
+      </IonButton>
+    );
+  };
+
+  const handleItemSelection = (index: number) => {
+    // @TODO - sdisalvo: add some logic for selection
+  };
+
+  const MenuItem = ({
+    index,
+    icon,
+    label,
+  }: {
+    index: number;
+    icon: string;
+    label: string;
+  }) => {
+    return (
+      <IonCol>
+        <div
+          onClick={() => handleItemSelection(index)}
+          data-testid={`menu-input-item-${index}`}
+          className="menu-input"
+        >
+          <IonIcon
+            slot="icon-only"
+            icon={icon}
+            color="primary"
+          />
+          <IonLabel>{label}</IonLabel>
+        </div>
+      </IonCol>
+    );
+  };
+
+  return (
+    <TabLayout
```

Comment:
@sdisalvo-crd i am not seeing ion-Grid layout https://ionicframework.com/docs/api/grid are we not using built-in components?